py/rcm_recoverer.py

```diff
@@ -37,43 +36,34 @@ def smart_open(filename, mode="r"):
             fh.close()
 
 
-def generate_rcm(reads_file_name, compressed_file_name, cliques_ids_file_name,
-                 out_file):
-    # Obtain read ids
-    with smart_open(reads_file_name, "r") as fh:
-        ids = [str(record.id) for record in SeqIO.parse(fh, "fasta")]
+def generate_rcm(compressed_rcm_file_name, clique_ids_file_name, out_file):
```

Comment:
Now we can rename `clique_ids_file_name` arg. And whole func too. For now it's just RCM composition.